Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -73,13 +73,16 @@ void JitArm64::Init()
 
 bool JitArm64::HandleFault(uintptr_t access_address, SContext* ctx)
 {
+  // Ifdef this since the exception handler runs on a separate thread on Mac OS X (ARM)
+#if !defined(__APPLE__) && !defined(_M_ARM_64)
```

Comment:
If I'm understanding the comment correctly this should be || instead of &&.

Reply:
It should actually be !(defined(__APPLE__) && defined(_M_ARM_64)) as we still want the fault handler on everything except arm macOS builds. It will be fixed in the next push.